<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainDialog</class>
 <widget class="QDialog" name="MainDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>777</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>LeoFEA</string>
  </property>
  <widget class="QScrollArea" name="scrollArea">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1101</width>
     <height>851</height>
    </rect>
   </property>
   <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="scrollAreaWidgetContents">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1099</width>
      <height>849</height>
     </rect>
    </property>
    <widget class="QPushButton" name="button_LoadLDR">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>220</y>
       <width>241</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Load LDR</string>
     </property>
    </widget>
    <widget class="QPushButton" name="button_startFEA">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>400</y>
       <width>241</width>
       <height>71</height>
      </rect>
     </property>
     <property name="text">
      <string>Run FEA</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>570</y>
       <width>90</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Postprocess</string>
     </property>
    </widget>
    <widget class="QPushButton" name="button_viewPNG">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>600</y>
       <width>241</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>View PNG</string>
     </property>
    </widget>
    <widget class="QPushButton" name="button_openParaview">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>640</y>
       <width>241</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Open Paraview</string>
     </property>
    </widget>
    <widget class="QPushButton" name="button_checkLimitValues">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>680</y>
       <width>241</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Check Limits</string>
     </property>
    </widget>
    <widget class="QPushButton" name="button_openLeoCAD">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>50</y>
       <width>241</width>
       <height>71</height>
      </rect>
     </property>
     <property name="text">
      <string>Open LeoCAD</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_13">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>160</y>
       <width>141</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>LDR-Dataset</string>
     </property>
    </widget>
    <widget class="QPushButton" name="button_ViewLDR">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>260</y>
       <width>241</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>View LDR</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>360</y>
       <width>521</width>
       <height>161</height>
      </rect>
     </property>
     <property name="title">
      <string>FEA Settings</string>
     </property>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>70</y>
        <width>111</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Analysis</string>
      </property>
     </widget>
     <widget class="QComboBox" name="combo_Analysis">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>70</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="editable">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>70</y>
        <width>131</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Max. Displacement</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>110</y>
        <width>101</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Load scale</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>30</y>
        <width>101</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Max. Force</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="textbox_gravityScale">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>110</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>1</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="textbox_maxForce">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>30</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>5</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="textbox_maxDisplacement">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>70</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>0.01</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_14">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>110</y>
        <width>51</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Contact</string>
      </property>
     </widget>
     <widget class="QComboBox" name="combo_Contact">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>110</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="editable">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QLineEdit" name="textbox_jobname">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>30</y>
        <width>171</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>71</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Jobname</string>
      </property>
     </widget>
    </widget>
    <widget class="QLabel" name="label_15">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>10</y>
       <width>141</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Model Design</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>30</y>
       <width>521</width>
       <height>121</height>
      </rect>
     </property>
     <property name="title">
      <string>Path settings</string>
     </property>
     <widget class="QPushButton" name="button_SelectPath_LeoCAD">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>20</y>
        <width>61</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Select</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_work_dir">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>80</y>
        <width>71</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Output FEA</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_12">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>50</y>
        <width>71</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Path Paraview</string>
      </property>
     </widget>
     <widget class="QPushButton" name="button_SelectPath_Paraview">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>50</y>
        <width>61</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Select</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="textbox_path_leoCAD">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>20</y>
        <width>321</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>C:/programs64/LeoCAD</string>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLineEdit" name="textbox_workdir">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>80</y>
        <width>321</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>C:/temp</string>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>71</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Path LeoCAD</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="textbox_path_paraview">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>50</y>
        <width>321</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>C:/programs64/Paraview5_12_RC/bin</string>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="button_SelectPath_OutputFEA">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>80</y>
        <width>61</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Select</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>170</y>
       <width>521</width>
       <height>161</height>
      </rect>
     </property>
     <property name="title">
      <string>Info LDR-Dataset</string>
     </property>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>70</y>
        <width>41</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Mass = </string>
      </property>
     </widget>
     <widget class="QLabel" name="label_16">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>141</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Filename = </string>
      </property>
     </widget>
     <widget class="QLineEdit" name="textbox_numberSegments">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>110</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="textbox_mass">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>70</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>110</y>
        <width>141</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Number of segments = </string>
      </property>
     </widget>
     <widget class="QLineEdit" name="textbox_FilenameLDR">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>30</y>
        <width>401</width>
        <height>31</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_17">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>70</y>
        <width>91</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Number of parts =</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="textbox_costs">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>110</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>110</y>
        <width>71</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Costs = </string>
      </property>
     </widget>
     <widget class="QLineEdit" name="textbox_numberParts">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>70</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
    </widget>
    <widget class="QLabel" name="label_18">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>340</y>
       <width>141</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>FEA</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox_4">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>550</y>
       <width>521</width>
       <height>211</height>
      </rect>
     </property>
     <property name="title">
      <string>Results</string>
     </property>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>10</y>
        <width>51</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Point</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>40</y>
        <width>141</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Max. tensile Force</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="textEdit_resultsMinForce_p">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>40</y>
        <width>141</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QTextEdit" name="textEdit_resultsMinForce">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>40</y>
        <width>71</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QTextEdit" name="textEdit_resultsMinForce_m">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>40</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QTextEdit" name="textEdit_resultsMaxForce_p">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>80</y>
        <width>141</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QTextEdit" name="textEdit_resultsMaxForce_m">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>80</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QTextEdit" name="textEdit_resultsMaxForce">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>80</y>
        <width>71</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>80</y>
        <width>171</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Max. compressive Force</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="textEdit_resultsMaxDisplacement">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>160</y>
        <width>71</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QTextEdit" name="textEdit_resultsMinDisplacement">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>120</y>
        <width>71</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QTextEdit" name="textEdit_resultsMaxDisplacement_p">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>160</y>
        <width>141</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QTextEdit" name="textEdit_resultsMinDisplacement_p">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>120</y>
        <width>141</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QTextEdit" name="textEdit_resultsMaxDisplacement_m">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>160</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QTextEdit" name="textEdit_resultsMinDisplacement_m">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>120</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>120</y>
        <width>141</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Min. Displacement</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>160</y>
        <width>141</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Max. Displacement</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
